### PR TITLE
JUPITER-42: override checkmark helix styles

### PR DESCRIPTION
**Pull request by anuritjain**
# Description
Checkmark is overriding the text "AWS Originated" under transaction type in the ChoiceTile
Jira: https://jira.rax.io/browse/JUPITER-42

Kapture 2021-01-06 at 21 51 33

### PR Checklist before asking for +2 from peers.
- [x] Did you include screenshots for changed UI functionality?
- [x] Did you add component tests for any new code?
- [x] Were you able to signup successfully with no console errors?
- [x] Have you deployed and validated your changes in staging?
- [x] Did you run the automated [Web-o-mates overnight tests](https://cq-portal.webomates.com/#/dashboard?product=95) on staging by clicking on the center circle in the CQ portal?
- [ ] Does this PR require more complete testing, such as a full regression before it is deployed?
- [ ] Have you documented any known deployment risks in this PR (If risk factors can not be mitigated a change request may need to be opened for this PR before it merges.)
### Before merging to master
- [x] I notified #jupiter-dev I am going to do a deployment before merging to master.

### CODE REVIEW

File: app/index.css [hunk 1]
```diff
@@ -224,4 +224,6 @@ hx-text-control[hx-dirty] > input[type="password"]:enabled:invalid, hx-text-cont
     text-align: center;
     margin: 10%;
 }
-
+hx-tile hx-icon[type="checkmark"]{
```
Comment (nicko-winner) [inline app/index.css:227]: It seems to help, but still gets covered at slightly lower resolutions (around 1024px).  I wonder if we just want to expand the size of the container at these lower resolutions so the margins are smaller. (see red area in image above)

Not that we want to go crazy making this a responsive application, but just ensuring a minimum canvas size for folks on their older 1024x768 screens might be helpful for making sure multi column items fit reasonably well. 

Though there are likely other solves that will work as well.

**Review by vaibhavdubey08 — APPROVED** [review]

File: app/index.css (hunk 1 above)
Comment (anuritjain) [reply app/index.css]: fixed. Please review once

File: app/containers/App.js [hunk 2]
```diff
@@ -29,7 +29,7 @@ export class App extends React.Component {
         <div id="app" className="u-flex-grow">
           <div id="stage" className="jupiter-content">
             <main role="main" id="content" className="main-body">
-              <div className="SignUp-container hxSpan-7-lg hxSpan-9-sm hxSpan-11-xs">
+              <div className="SignUp-container hxSpan-8-lg hxSpan-9-sm hxSpan-10-md hxSpan-11-xs">
```
Comment (abrande) [inline app/containers/App.js:32]: It looks like there will be overlap either way when the screen is smaller, but it looks like `hxSpan-8-sm` overlaps less between certain screen sizes

**Review by abrande — APPROVED** [review]

File: app/containers/App.js (hunk 2 above)
Comment (anuritjain) [reply app/containers/App.js]: added  `hxSpan-8-sm` and now looks fine when screen is smaller.

**Review by abrande — APPROVED** [review]

**Pull request merged**